middleware.js

```diff
@@ -0,0 +1,12 @@
+// middleware.ts
```

Comment:
Where is this file used? If test or example only, can you move it inside those folder.

```suggestion
// middleware.ts
'use strict'
```

Reply:
Unfortunately I can't move it to another place because the changes of Next.js 13 are based on a single file which must be at the same level as the pages or app folder

According to docs : 
> Create a middleware.ts (or .js) file at the root or in the src directory (same level as your pages)

https://nextjs.org/docs/advanced-features/middleware